### PR TITLE
Fixed auto complete pr coments

### PR DESCRIPTION
**Pull request by Ankit-Keshari-Vituity**
Changes: 

1. Worked on cancelModalVisible and setCancelModalVisible.
2. Worked on text.length > 0.
3. Used IconStyleType.ACCENT 

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: datahub-web-react/src/app/shared/tags/AddTagTermModal.tsx [hunk 1]
```diff
@@ -74,49 +68,56 @@ export default function AddTagTermModal({
...
+            tagSearch({
                 variables: {
                     input: {
-                        types: [type],
-                        query,
-                        limit: 25,
+                        type: EntityType.Tag,
+                        query: text,
+                        start: 0,
+                        count: 5,
```
Comment (jjoyce0510) [inline datahub-web-react/src/app/shared/tags/AddTagTermModal.tsx:89]: One last thing -- Let's increase this to 10

Comment (jjoyce0510) [inline datahub-web-react/src/app/shared/tags/AddTagTermModal.tsx:89]: One last thing -- Let's increase this to 10

Comment (jjoyce0510) [inline datahub-web-react/src/app/shared/tags/AddTagTermModal.tsx:89]: One last thing -- Let's increase this to 10

**Review by shirshanka — APPROVED** [review]
LGTM

**Pull request merged**